app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<h2>Sign up form</h2>
```

Comment:
- [ ] Great job! However, it would be better if the views in devise for the registration and login matches the wireframes and their styling.You can take a look at the wireframes in the [sneak peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) of the project. Kindly implement that.
- This is your current style for registration and login pages :point_down: 
![Screenshot from 2022-05-24 22-29-18](https://user-images.githubusercontent.com/67087664/170097280-35117dcf-e050-43c2-88cb-2b69e699a7dd.png)
![Screenshot from 2022-05-24 22-24-56](https://user-images.githubusercontent.com/67087664/170097285-12fbe9af-24ca-4eaa-b810-073fea0609fb.png)

Reply:
The design now looks like the wireframe provided. Great job @HAMMAS-SALEEM 👍🏽